content/enable-hibernation.md

```diff
@@ -29,7 +29,11 @@ Hibernation is not currently enabled on Pop!\_OS by default. There are several l
 - Additional drive I/O used on Solid State Drives (SSDs)
 - Depending on alloted RAM in a system, and size of NVMe/SSD drive(s), hibernation could add notable delay on resuming a hibernation session.
 
-However, these limitations aside, hibernation is an often requested feature for Pop!\_OS. The purpose of this article is to provide steps for enabling hibernation for any users that would like it.
+However, these limitations aside, hibernation is an often requested feature for Pop!\_OS, and there are some real benefits to it. For example, hibernation (when compared to always using suspend) can improve the longevity of your battery - full charge/discharge cycles can be hard on your battery and shorten its life. Since hibernation completely shuts off power to the device, it can prolong the lifetime of your battery. Additionally, there are some security benefits. While your computer is suspended, the decryption key for your data volume remains in memory, and your device is vulnerable to certain physical attacks that can extract the key from RAM. If the device is hibernated, and the hibernation swap space is encrypted, the drive must be decrypted to obtain the key, which would be impossible without already knowing the key (this is as secure as a normal, full shutdown).
```

Comment:
> and the hibernation swap space is encrypted

I believe this is correct, given I have to enter my password for the computer to resume after it has been hibernated (and I assume this is why the guide moves from a swap partition to a swap volume in the LVM setup), but it's probably worth someone confirm that this is true.

Reply:
Pretty sure https://wiki.archlinux.org/title/Dm-crypt/Swap_encryption#With_suspend-to-disk_support confirms it, but again, good to have someone verify security claims.

Reply:
The change between hibernation and suspend changes the security of the system not at all. Pop!OS uses a encrypted swap at all times. Hibernation-than-suspend will still have any encryption keys in memory on suspend. Battery life is unlikely to be changed by hibernation unless you are only rarely using the device and ended up with many additional charging cycles--in which case you should be powering off the device anyway. Hibernation does increase your ware on NVMe/SSD drives that will ware out the hardware over just using suspend.